h2o-hadoop/h2o-mapreduce-generic/src/main/java/water/hadoop/h2odriver.java

```diff
@@ -1075,6 +1129,37 @@ private void addMapperConf(Configuration conf, String name, String value, byte[]
     mapperConfLength++;
   }
 
+  private static ServerSocket bindCallbackSocket() throws IOException {
+    Exception ex = null;
+    int securityExceptionCount = 0;
+    ServerSocket result = null;
+    for (int p = driverCallbackPortRange.from; p <= driverCallbackPortRange.to; p++) {
+      ServerSocket ss = new ServerSocket();
+      ss.setReuseAddress(true);
+      InetSocketAddress sa = new InetSocketAddress(driverCallbackIp, p);
+      try {
+        ss.bind(sa, p); // FIXME: ??? why is the 'backlog' parameter set this way (to a value of the port number)?
```

Comment:
I think that's mistake. I do not see any relation between backlog and port number...